googlesearch/__init__.py

```diff
@@ -33,6 +33,10 @@
 import time
 import ssl
 
+#suggested imports
+from datetime import date
```

Comment:
Not sure why we need this, the googlesearch module doesn't directly use them

Reply:
both imports are unused, shouldn't be introduced in the code.